samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -624,6 +652,10 @@ mfxStatus CEncodingPipeline::InitMfxEncParams(sInputParams *pInParams)
 
         m_EncExtParams.push_back((mfxExtBuffer *)&m_CodingOption3);
     }
+    else if (m_CodingOption3.RepackMaxIFrameSize || m_CodingOption3.RepackMaxPFrameSize || m_CodingOption3.RepackMaxBFrameSize)
```

Comment:
looks like this block must be merged with previous. Otherwise the buffer pushed here will contain many unset fields.

Reply:
Repack max frame size parameters are independent with other parameters in m_CodingOption3, so I don't think we should merge this with previous.